docs/guides/image-update.md

```diff
@@ -594,286 +594,191 @@ some common examples for the most popular cloud provider docker registries.
 
 ### AWS Elastic Container Registry
 
-The registry authentication credentials for ECR expire every 12 hours.
-Considering this limitation, one needs to ensure the credentials are being
-refreshed before expiration so that the controller can rely on them for
-authentication.
+We can create a deployment that frequently refreshes an image pull secret into our desired namespace.
 
-The solution proposed is to create a cronjob that runs every 6 hours which would
-re-create the `docker-registry` secret using a new token.
+Create a directory in your control repository and save this `kustomization.yaml`:
```

Comment:
What is a "control repository"? This seems to be the first mention (in this document at least).

Reply:
Should be something along the lines of the repo you bootstrapped flux wiith